code/src/index.css

```diff
@@ -11,3 +13,201 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* survey.js (mainfile) */
+
+header{
+  background-image: url('./images/background.jpg');
+  height: 13vh;
+  position: relative;
+  }
+
+.maincontainer{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 85vh;
+  width: 100vw;
+  font-family: Spartan;
+}
+
+.maintitle{
+  margin-top: 30px;
+  font-family: Fascinate;
+  margin-bottom: 0;
+  font-size: 28px;
+  text-align: center;
+  width: 100%;
+}
+
+form{
+  display: flex;
+  flex-wrap: wrap;
+  margin: 40px 0;
+}
+
+.formelement{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.questioncontainer{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 100%;
+}
+
+p{
+  font-size: 12px;
+  text-align: center;
+  margin: 10px 0 5px 0;
+  width: 100%;
+  line-height: 20px;;
+}
```

Comment:
In general, it  can be good to create a class for the paragraphs and style class in order to make it easier to handle for larger projects. This approach is still possible to work with as long as the project is quite small :)